build.gradle

```diff
@@ -198,22 +198,22 @@ dependencies {
 
   implementation group: 'com.microsoft.azure', name: 'azure-spring-boot-starter', version: versions.azureSpringBoot
   implementation group: 'org.springframework.boot', name: 'spring-boot-starter-mail', version: versions.springBootMail
+  implementation group: 'com.microsoft.azure', name: 'applicationinsights-spring-boot-starter', version: '2.4.0'
 
   implementation group: 'com.opencsv', name: 'opencsv', version: versions.openCsv
   implementation group: 'net.lingala.zip4j', name: 'zip4j', version: versions.zip4j
   implementation group: 'net.jodah', name: 'failsafe', version: versions.failsafe
+  implementation group: 'commons-io', name: 'commons-io', version: '2.11.0'
 
   // Reform libraries
-  implementation group: 'uk.gov.hmcts.reform', name: 'logging', version: versions.reformLogging
-  implementation group: 'uk.gov.hmcts.reform', name: 'logging-appinsights', version: versions.reformLogging
   implementation('uk.gov.hmcts.reform:properties-volume-spring-boot-starter:0.1.0') {
     exclude group: 'org.springframework.boot', module: 'spring-boot-starter-web'
   }
   implementation group: 'com.jcraft', name: 'jsch', version: versions.jcraft
 
-  implementation group: 'uk.gov.hmcts.reform.mi', name: 'mi-core-lib', version: versions.miCoreLib
-  implementation group: 'uk.gov.hmcts.reform', name: 'logging', version: versions.reformLogging
-  implementation group: 'uk.gov.hmcts.reform', name: 'logging-appinsights', version: versions.reformLogging
+  implementation group: 'com.github.hmcts', name: 'mi-core-lib', version: versions.miCoreLib
+  //  implementation group: 'com.github.hmcts.java-logging', name: 'logging', version: versions.reformLogging
```

Comment:
Logging has been moved to jitpack but the versions available only support Spring Boot 2.5.3+
This has major breaking changes with extraction-service so we will use an older implementation group: 'com.microsoft.azure', name: 'applicationinsights-spring-boot-starter', version: '2.4.0' instead for the appinsights telemetry.